cluster/addons/dns/kube2sky/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scratch
```

Comment:
Why do you need dynamic libs?  That's sooo 2012.



On Fri, May 15, 2015 at 10:24 AM, Victor Marmol <notifications@github.com>
wrote:

> In cluster/addons/dns/kube2sky/Dockerfile
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8322#discussion_r30427706>
> :
>
> > @@ -1,4 +1,4 @@
> > -FROM scratch
>
> +1 we found that busybox does not have most of the dynamically linked
> libraries Go uses.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8322/files#r30427706>
> .
>